tests/lit-tests/pic-level.ispc

```diff
@@ -0,0 +1,21 @@
+// RUN: %{ispc} --target=host --nostdlib --nowrap %s -o - --emit-llvm-text 2>&1 | FileCheck %s --check-prefix=CHECK-NOPIC
+// RUN: %{ispc} --target=host --nostdlib --nowrap %s -o - --emit-llvm-text --pic 2>&1 | FileCheck %s --check-prefix=CHECK-PIC
+// RUN: %{ispc} --target=host --nostdlib --nowrap %s -o - --emit-llvm-text --PIC 2>&1 | FileCheck %s --check-prefix=CHECK-BIGPIC
+
+// Note: in LLVM 16 the merge behaviour of PIC flags for Modules changed.
+// Before, the merge value was the maximal. We are merging the flags from the
+// user module, the builtin-cpp module and the target module. So, the PIC level
+// of the resulted code depends not only on the user flags, but also on the
+// flags that we build builtin and target modules.
+// REQUIRES: LLVM_16_0+
```

Comment:
It should not run with target OS windows, right?

Reply:
Ok, I have fixed that. But there is no actual difference in the LLVM IR level between Linux and Windows for "PIC Level" values.